src/Symfony/Component/Yaml/Parser.php

```diff
@@ -359,6 +299,31 @@ public function parse($value, $flags = 0)
             mb_internal_encoding($mbEncoding);
         }
 
+        if (array_key_exists('<<', $data)) {
```

Comment:
We should be able to detect if someone uses `null` and throw a proper exception.